static/src/javascripts/projects/common/modules/navigation/edition-picker.js

```diff
@@ -1,84 +1,41 @@
 define([
-    'common/utils/fastdom-promise',
-    'common/utils/$'
+    'fastdom',
+    'qwery'
 ], function (
-    fastdomPromise,
-    $
+    fastdom,
+    qwery
 ) {
-    var dropdown = $('.js-edition-picker-dropdown');
-    var button;
+    function editionPickerClickHandler(event) {
+        event.stopPropagation();
+        var button = event.target;
+        var editionPickerDropdown = qwery('.js-edition-picker-dropdown')[0];
```

Comment:
Will the edition picker dropdown always be there? If not, we should wrap line 25 and 34 in` if (editionPickerDropdown) ...` or use a try catch.